src/custom_modules/vipps.ts

```diff
@@ -201,7 +201,7 @@ module.exports = {
     };
 
     let initiateRequest = await request.post({
-      uri: `https://${config.vipps_api_url}/ecomm/v2/payments`,
+      uri: `https://${config.vipps_api_url}/ecomm/v3/payments`,
```

Comment:
I think ecomm v3 API [might not exist](https://developer.vippsmobilepay.com/docs/APIs/ecom-api/vipps-ecom-api/)

Reply:
Ohh good catch! I will change that!